en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/07-security.markdown

```diff
@@ -169,27 +175,47 @@ configuration (`elasticsearch.yml`) used in this guide (with the Elasticsearch 6
 example commented out):
 
 ```yaml
-# For Elasticsearch 7
+# For Elasticsearch 7.3/7.4
 cluster.name: LiferayElasticsearchCluster
+
+# X-Pack Security
 xpack.security.enabled: true
+
+## TLS/SSL settings for Transport layer
 xpack.security.transport.ssl.enabled: true
-xpack.security.transport.ssl.verification_mode: certificate
-xpack.security.transport.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.transport.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.transport.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.transport.ssl.verification_mode: certificate 
+xpack.security.transport.ssl.key: /path/to/localhost.key
+xpack.security.transport.ssl.certificate: /path/to/localhost.crt
+xpack.security.transport.ssl.certificate_authorities : ["/path/to/ca.crt"]
+
+# TLS/SSL settings for HTTP layer
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.http.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.http.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.http.ssl.verification_mode: certificate 
+xpack.security.http.ssl.key: /path/to/localhost.key
+xpack.security.http.ssl.certificate: /path/to/localhost.crt
```

Comment:
Same questions a above. Is it no longer necessary to put these in `ES_HOME/config`?

Reply:
I just copy-pasted the pattern from the last line where you didn't have to ES config home. You can add that part back to each property.